contracts/src/libraries/EigenDARollupUtils.sol

```diff
@@ -18,7 +18,7 @@ library EigenDARollupUtils {
     // STRUCTS
     struct BlobVerificationProof {
         uint32 batchId;
-        uint8 blobIndex;
+        uint256 blobIndex;
```

Comment:
uint32 should be fine.

also a regression test would be good as a pending issue

Reply:
yeah, offchain is also uint32: https://github.com/Layr-Labs/eigenda/blob/master/api/proto/disperser/disperser.proto#L190